terraform/hcl/applicable/req_sw_terraform_aws_cloudfront_tls_only.tf

```diff
@@ -0,0 +1,22 @@
+resource "aws_cloudfront_distribution" "vulnerable_example" {
+  default_cache_behavior {
+    viewer_protocol_policy = "allow-all"
+
+    target_origin_id = "dummy"
+    allowed_methods  = ["dummy"]
+    cached_methods  = ["dummy"]
+  }
+
+  enabled             = true
+  origin {
+    origin_id = "dummy"
+    domain_name = "dummy"
+  }
+  restrictions {
+    geo_restriction {
+        restriction_type = "none"
+    }
+  }
+  viewer_certificate {
+  }
+}
```

Comment:
[comment]: <> (FrogbotReviewComment)


## 🛠️ Infrastructure as Code Vulnerability
<div align='center'>

| Severity                | Finding                  |
| :---------------------: | :-----------------------------------: |
| ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | viewer_protocol_policy='allow-all' was detected |

</div>

<details>
<summary> <b>Full description</b> </summary>
<br>

Resource `aws_cloudfront_distribution` should have `viewer_protocol_policy=https-only` or `viewer_protocol_policy=redirect-to-https`

Vulnerable example -
```
resource "aws_cloudfront_distribution" "vulnerable_example" {
  default_cache_behavior {
    viewer_protocol_policy = "allow-all"
  }
}
```

Secure example -
```
resource "aws_cloudfront_distribution" "secure_example" {
  default_cache_behavior {
    viewer_protocol_policy = "https-only"
  }
}
```

</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>